src/Config/Main.hs

```diff
@@ -178,6 +179,7 @@ defaultConfig =
       , _mvMailListOfAttachmentsKeybindings = mailAttachmentsKeybindings
       , _mvOpenWithKeybindings = openWithKeybindings
       , _mvPipeToKeybindings = pipeToKeybindings
+      , _mvMailcap = Map.fromList [(("text", "html"), "elinks -force-html")]
```

Comment:
If we want to reflect what mailcap is capable of, we need to support "any" (i.e. wildcard) for the subtype.  To mirror what *purebred-email* does we would have `(CI B.ByteString, Maybe (CI B.ByteString))`.  But really we could just use the type [(ContentType -> Bool, String {- comand -}], and traverse this list until we get a hit.  That means we could use `Data.MIME.matchContentType` to easily construct the predicate function:
```haskell
matchContentType
  :: CI B.ByteString         -- ^ type
  -> Maybe (CI B.ByteString) -- ^ optional subtype
  -> ContentType
  -> Bool
```

Reply:
Yeah true. I hoped with the Map it would have been easier, but really there isn't any benefit at the moment. Cheers

Reply:
Alright updated. However, I didn't go with the:
```Haskell
[(ContentType -> Bool, String {- comand -}]
```
but used instead:
```Haskell
[(CI B.ByteString, Maybe (CI B.ByteString))]
```
Reason being that I can't see the point of taking a function with me for the first part of the tuple. Misunderstanding maybe?

Reply:
Just realized what you meant. Hm happy to update the patch or merge of it's not a deal breaker.

Reply:
Don't merge yet, I'd like to play with it, but yeah `[(type, Maybe subtype)]` is fine with me for now.